package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nasa-jpl/seq-json-schema",
-  "version": "1.0.21",
```

Comment:
😉

Reply:
We probably want to up the version to 1.0.22

Reply:
Agree -- this is a backward-compatible added feature, so I think it merits a minor version increment. Sequences that leverage this added feature won't be backward compatible.